### PR TITLE
Adds freeze functionality on columns

### PR DESCRIPTION
**Pull request by bkovacik**
Helps to close KoddiDev/HPA#6941.

### CODE REVIEW

**Review by spencersmith — APPROVED** [review]

**Review by mcross1882 — APPROVED** [review]
There are some things in here we should probably be accessing via the query selector instead of using `children()` + `eq()` but its good to launch if its working properly.

**Comment by kyrozetera** [comment]
@mcross1882 I agree, it would make it a lot clearer.

File: dist/reporttable.js [hunk 1]
```diff
@@ -14,155 +14,239 @@
...
+        this.container.on("colFreezeToggle", function (event, index) {
+            var widths = that.getOuterWidthsFromFirstRow();
+            var headerHeight = that.getHeightFromHeader();
+            var rowHeight = that.getHeightFromFirstRow();
+            var frozenIndex = that.frozenIndices.indexOf(index);
+
+            frozenIndex == -1 ? 
+                that.frozenIndices.push(index) : that.frozenIndices.splice(frozenIndex, 1);
+
+            $(".reporttable_header, .koddiTableGraph, .reporttable_footer").find("tr").each(function () {
```
Comment (kyrozetera) [inline dist/reporttable.js:65]: We should restrict this selector to this container so it doesn't affect other tables on the same page.

Comment (kyrozetera) [reply dist/reporttable.js:65]: There are several more instances where we use global selectors, so we should reduce the scope of those as well.

Comment (kyrozetera) [reply dist/reporttable.js:65]: Also, `.koddiTableGraph` is not part of this repo and is specific to the original table object, so we should change that selector. Maybe to select the second table of the container?

**Pull request merged**